site/community/index.md

```diff
@@ -0,0 +1,28 @@
+---
+nav_id: MAIN_COMMUNITY
+layout: default
+
+title: Community
+---
+
+{% contentfor header %}
+
+# Open Policy Agent
+{: .opa-header--title}
+
+Under active development.
+{: .opa-header--subtitle}
+
+Open Policy Agent (OPA) is an open source policy engine. Contributions are always welcome. <span class="opa-keep-it-together">For information about contributing, see [CONTRIBUTING.md](https://github.com/open-policy-agent/opa/blob/master/CONTRIBUTING.md){: .opa-header--link}.</span>
+{: .opa-header--text}
+
+{% endcontentfor %}
+
+{% contentfor body %}
+
+  * [{% img '{{assets["community-slack.svg"].logical_path}}' alt:'Slack: community chat' %}](https://openpolicyagent.slack.com/)
```

Comment:
We need to deploy a bot to send invites to people who want to use the OPA slack organization.

I think we should wait for the bot before including the link here and in the footer.